docs/reference/agent-api/debug.md

```diff
@@ -0,0 +1,76 @@
+# Debug
+
+## Get debug logs
+
+### Code samples
+
+```shell
+curl http://127.0.0.1:2113/debug/logs
```

Comment:
Worthwhile referring to the config flag which defines this address (`CODER_AGENT_DEBUG_ADDRESS`), I think.

Reply:
Sounds good, I can make this change